### DISCUSSION TITLE
Using laravel mix

### DISCUSSION THREAD

**Discussion by sys-auditing** [PrismJS/prism#2987] [Q&A]
Hi, i would like to use PrismJS with laravel mix (webpack).
How can i achieve that ?

Thank you

**Comment by RunDevelopment** [comment]
We make a library, not the framework. So I can't really help you with that.

However, following [their documentation](https://laravel.com/docs/8.x/mix) and [our documentation](https://prismjs.com/#basic-usage) should give you all the information you need. Also, Laravel seems to be using Prism on their website, so looking up the website's source (if available) might be a good idea.

If that doesn't help, I recommend asking this question on a Laravel discussion page (if any) or StackOverflow.

**Reply by sys-auditing** [reply]
i see, 
thank you.

**Comment by sys-auditing** [comment]
I have just figured out  the issue.

```// bootstrap.js```
```js
  window.Prism = require('prismjs');
  require('prismjs/components/prism-markup-templating');
  require('prismjs/components/prism-bash');
  require('prismjs/components/prism-php');

  Prism.highlightAll();
  ```
  
  ```//app.scss```
  ```css
  @import "~prismjs/themes/prism.css";
  ```
  
  Thank you
